x-pack/plugins/security_solution/public/flyout/left/context.tsx

```diff
@@ -17,6 +27,10 @@ export interface LeftPanelContext {
    * Name of the index used in the parent's page
    */
   indexName: string;
+  /**
+   * Retrieves searchHit values for the provided field
+   */
+  getFieldsData: (field: string) => unknown | unknown[];
```

Comment:
`useGetFieldsData` has a type called `GetFieldsData` in 'x-pack/plugins/security_solution/public/common/hooks/use_get_fields_data.ts'

Reply:
yup a lot cleaner thanks!